sros2/sros2/keystore/_keystore.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2020 Canonical Ltd
+# Copyright 2016-2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import pathlib
+
+from cryptography import x509
+
+import lxml
+
+from sros2 import _utilities
+import sros2.errors
+from sros2.policy import get_transport_default, get_transport_schema
+
+
+_KS_ENCLAVES = 'enclaves'
+_KS_PUBLIC = 'public'
+_KS_PRIVATE = 'private'
+_DEFAULT_COMMON_NAME = 'sros2testCA'
```

Comment:
How about this instead,
```suggestion
_DEFAULT_COMMON_NAME = 'sros2defaultCA'
```

Reply:
Ah heck, let's go all the way with `sros2CA`. Pushed.